### ISSUE TITLE
placeholder not working in IE after going by turbo-link

### ISSUE THREAD

**Issue by kritik** [rails/turbolinks#455]
First time page load  - ok but after going by link placeholder becomes value.   
IE 10 ja 11 - not working. 
Firefox and Chrome are ok. 

gem versions: 2.5.3 and repo master HEAD. Testing code:
```html
<!DOCTYPE html>
<head>
   <script data-turbolinks-track="" src="/assets/turbolinks.js?body=1"></script>
</head>
<body class="page-index">
<a href="turbolinks.html?a=b">link</a>
<textarea placeholder="test"></textarea>
</body>
</html>
```

**Comment by inspire22** [comment]
I'm having the same issue, it is driving me nuts!

I came across the "vue" project having the same issue, they said: 

     This is actually a bug in IE10/11: when a <textarea> with placeholder attribute is cloned, the clone's value is set to the original's placeholder. I implemented a workaround in Vue's template parser to deal with this.

**Comment by inspire22** [comment]
here's the workaround I'm using until this is fixed - I call this at page:change  (coffeescript)

    fix_ie10_placeholder = ->
      $('textarea').each ->
        if $(@).val() == $(@).attr('placeholder')
          $(@).val ''

**Comment by kritik** [comment]
Yes, we thought about the same fix. But it would be nice if turbolinks fixes this behaviour out of the box.

**Comment by yui-knk** [comment]
I come up with same bug.
See https://connect.microsoft.com/IE/feedback/details/811408/ie10-11-a-textareas-placeholder-text-becomes-its-actual-value-when-using-innertext-innerhtml-or-outerhtml

**Comment by Thibaut** [comment]
This has been fixed in master by 86518e10ea503c691e9f24ac2621a50602696bb4 and d67a2d5ad8680ce4a653242840d9300206c2a997.

**Comment by yui-knk** [comment]
@Thibaut  Thanks !!!!
